src/Tgstation.Server.Host/Components/Watchdog/WatchdogBase.cs

```diff
@@ -121,7 +121,7 @@ public WatchdogStatus Status
 		/// <summary>
 		/// The <see cref="IEventConsumer"/> that is not the <see cref="WatchdogBase"/>
 		/// </summary>
-		readonly IEventConsumer eventConsumer;
+		public IEventConsumer eventConsumer;
```

Comment:
```suggestion
		protected IEventConsumer EventConsumer { get; }
```
And move it up to under line 84